### PR TITLE
Add `zh_HK` Locale Language Configuration File

### PR DESCRIPTION
**Pull request by Lunari8546**
Added translations for Chinese Traditional (HKSAR) (Cantonese) to support current HarTex nightly.

### CODE REVIEW

File: langcfgs/zh_HK.langcfg [hunk 1]
```diff
@@ -22,3 +22,19 @@
 ; with HarTex. If not, see <https://www.gnu.org/licenses/>.
 
 LanguageIdentifier: zh_HK
+
+; Section: Global Plugin, About Command
+GlobalPlugin.AboutCommand.EmbedDescription: 由 HarTex 團隊維護及開發, HarTex 係一個針對 Discord 高效管理去進行咗優化嘅 Bot.
```
Comment (HTGAzureX1212) [inline langcfgs/zh_HK.langcfg]: could probably reword this sentence: `HarTex 係一個由 HarTex 團隊維護及開發嘅 Discord Bot，主要針對 Discord server 高效管理去進行咗優化。`?

File: langcfgs/zh_HK.langcfg [hunk 2]
```diff
@@ -22,3 +22,19 @@
...
+GlobalPlugin.AboutCommand.EmbedBotVersionFieldName: Bot 嘅版本
+GlobalPlugin.AboutCommand.EmbedWhitelistedGuildsFieldName: Whitelist 咗嘅 Server
+
+; Section: Global Plugin, Ping Command
+GlobalPlugin.PingCommand.InitialResponse: Hello! 有咩幫到你? :eyes:
```
Comment (HTGAzureX1212) [inline langcfgs/zh_HK.langcfg]: additionally translate `Hello!` phrase as well.

File: langcfgs/zh_HK.langcfg [hunk 3]
```diff
@@ -22,3 +22,19 @@
...
+GlobalPlugin.PingCommand.MillisecondUnit: 毫秒
+
+; Section: Global Plugin, Source Command
+GlobalPlugin.SourceCommand.PreRepositoryUriMessage: 個 Bot 嘅源代碼:
+
+; Section: Global Plugin, Team Command
+GlobalPlugin.TeamCommand.EmbedTitle: HarTex 團隊
+GlobalPlugin.TeamCommand.EmbedGlobalAdminAndLeadDevFieldName: Admin & 首席 Dev
```
Comment (HTGAzureX1212) [inline langcfgs/zh_HK.langcfg]: possible better translation: `總管理員 同埋 首席開發員`?

Comment (Lunari8546) [reply langcfgs/zh_HK.langcfg]: "總管理員" seems not too cantonese, but I am fine with "同埋 首席開發員".

Comment (HTGAzureX1212) [reply langcfgs/zh_HK.langcfg]: agreed

File: langcfgs/zh_HK.langcfg (hunk 1 above)
Comment (Lunari8546) [reply langcfgs/zh_HK.langcfg]: yea sure, LGTM.

File: langcfgs/zh_HK.langcfg (hunk 2 above)
Comment (Lunari8546) [reply langcfgs/zh_HK.langcfg]: You can decide on this.

File: langcfgs/zh_HK.langcfg (hunk 3 above)
Comment (Lunari8546) [reply langcfgs/zh_HK.langcfg]: btw, I think before and after "同埋" should not contains spacebar.

Comment (HTGAzureX1212) [reply langcfgs/zh_HK.langcfg]: Indeed, spaces around that seem to be unnecessary

**Review by HTGAzureX1212 — CHANGES_REQUESTED** [review]
please make the requested changes as with the previous conversations (and add commits to the branch)

**Comment by Lunari8546** [comment]
yea ok

**Review by HTGAzureX1212 — APPROVED** [review]

**Pull request merged**